components/lib/dropdown/DropdownPanel.js

```diff
@@ -76,7 +76,7 @@ export const DropdownPanel = React.memo(
             }
         };
 
-        const createGroupChildren = (optionGroup, style) => {
+        const createGroupChildren = (optionGroup, style, passedOptionsCount) => {
```

Comment:
```suggestion
        const createGroupChildren = (optionGroup, style, index) => {
```

Call `passedOptionsCount` just `index`